rsget_lib/src/plugins/vlive.rs

```diff
@@ -0,0 +1,180 @@
+use crate::Streamable;
+use regex::Regex;
+
+use stream_lib::StreamType;
+
+use crate::utils::downloaders::DownloadClient;
+
+use crate::utils::error::RsgetError;
+use crate::utils::error::StreamError;
+
+#[derive(Debug, Clone)]
+pub struct Vlive {
+    client: DownloadClient,
+    url: String,
+    title: String,
+    author: String,
+    video_url: Option<String>,
+    // TODO FOR ERK: This field is currently unused. This is due to Rsgets design being too focused on making plugin
```

Comment:
Maybe this should be moved into a seperate issue about redesign.

Reply:
Yes, I do admit a long rant in comments is perhaps not the best form to communicate issues.